### PR TITLE
[OSCD] win_visual_basic_compiler.yml added

### PR DESCRIPTION
**Pull request by esebese**
Parent-child relation defined for suspicious use cases.

### CODE REVIEW

File: rules/windows/process_creation/win_visual_basic_compiler.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Visual Basic Command Line Compiler Usage
+id: 7b10f171-7f04-47c7-9fa2-5be43c76e535
+status: experimental
+description: Detects successful code compilation via Visual Basic Command Line Compiler that utilizes Windows Resource to Object Converter.
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Vbc/
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_visual_basic_compiler.yml:7]: Hello @esebese !
Could you please apply [this](https://github.com/Neo23x0/sigma/pull/1062#discussion_r502062289) recommendation here as well?

File: rules/windows/process_creation/win_visual_basic_compiler.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+    - attack.defense_evasion
+    - attack.t1027.004
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentImage|endswith: '*\vbc.exe'
+        Image|endswith: '*\cvtres.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_visual_basic_compiler.yml:18]: could you please remove "`*`" from the `'*\vbc.exe'` and `'*\cvtres.exe'`, as it already declared by the `endswith`?

Comment (esebese) [reply rules/windows/process_creation/win_visual_basic_compiler.yml]: fixed, thanks

**Comment by yugoslavskiy** [comment]
Hello @esebese! The rule looks perfect. Thank you very much for your continuous work!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**